interface/globals.php

```diff
@@ -498,7 +498,7 @@ function GetCallingScriptName()
 //
 $top_bg_line = ' bgcolor="#dddddd" ';
 $GLOBALS['style']['BGCOLOR2'] = "#dddddd";
-$logocode = "<img class='img-fluid' src='" . $GLOBALS['OE_SITE_WEBROOT'] . "/images/login_logo.gif' />";
+$logocode = "<img class='img-responsive' src='" . $GLOBALS['OE_SITE_WEBROOT'] . "/images/login_logo.gif' />";
 // optimal size for the tiny logo is height 43 width 86 px
```

Comment:
I just curious, why we change the `img-fluid` to `img-responsive`?

Reply:
Fluid attempts to get the width of the image to 100% while `img-responsive` tends to not do that

Reply:
Usually I just go back and forth between the two and see which one works better